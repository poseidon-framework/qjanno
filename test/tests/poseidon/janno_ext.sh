qjanno "SELECT * FROM data/Schmid_2028/Schmid_2028.janno"