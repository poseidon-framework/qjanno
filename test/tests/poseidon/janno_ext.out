.------------------------------------.-------------.------------.-------------.
|            source_file             | Genetic_Sex | Group_Name | Poseidon_ID |
:====================================:=============:============:=============:
| data/Schmid_2028/Schmid_2028.janno | F           | POP1       | XXX001      |
| data/Schmid_2028/Schmid_2028.janno | F           | POP2       | XXX002      |
'------------------------------------'-------------'------------'-------------'
